README.yaml

```diff
@@ -20,25 +20,24 @@ copyrights:
     year: "2020"
 
 # Canonical GitHub repo
-github_repo: cloudposse/terraform-example-module
+github_repo: cloudposse/terraform-tfe-cloud-infrastructure-automation
 
 # Badges to display
 badges:
   - name: "Latest Release"
-    image: "https://img.shields.io/github/release/cloudposse/terraform-example-module.svg"
-    url: "https://github.com/cloudposse/terraform-example-module/releases/latest"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-tfe-cloud-infrastructure-automation.svg"
+    url: "https://github.com/cloudposse/terraform-tfe-cloud-infrastructure-automation/releases/latest"
   - name: "Slack Community"
     image: "https://slack.cloudposse.com/badge.svg"
     url: "https://slack.cloudposse.com"
   - name: "Discourse Forum"
     image: "https://img.shields.io/discourse/https/ask.sweetops.com/posts.svg"
     url: "https://ask.sweetops.com/"
 
-# List any related terraform modules that this module may be used with or that this module depends on.
 related:
-  - name: "terraform-null-label"
-    description: "Terraform module designed to generate consistent names and tags for resources. Use terraform-null-label to implement a strict naming convention."
-    url: "https://github.com/cloudposse/terraform-null-label"
+  - name: "terraform-kubernetes-tfc-cloud-agent"
+    description: "Installs the Terraform Cloud Agent on an existing Kubernetes cluster."
+    url: "https://github.com/cloudposse/terraform-kubernetes-tfc-cloud-agent"
 
 # List any resources helpful for someone to get started. For example, link to the hashicorp documentation or AWS documentation.
 references:
```

Comment:
Terraform Cloud - https://app.terraform.io/signup/account

Reply:
Terraform Enterprise/Cloud Docs - https://registry.terraform.io/providers/hashicorp/tfe/latest/docs

Reply:
Drop the documentation on `random_integer` resource, and stuff about publishing a module, and module structure. that's all boilerplate from the example.

Reply:
Done and done.